apis/hive/v1/clusterdeployment_types.go

```diff
@@ -401,6 +419,11 @@ const (
 	// AWSPrivateLinkFailedClusterDeploymentCondition is true controller fails to setup private link access
 	// for the cluster.
 	AWSPrivateLinkFailedClusterDeploymentCondition ClusterDeploymentConditionType = "AWSPrivateLinkFailed"
+
+	// These are conditions that are copied from ClusterInstall on to the ClusterDeployment object.
+	ClusterInstallFailedClusterDeploymentCondition          ClusterDeploymentConditionType = "ClusterInstallFailed"
```

Comment:
Today the condition is ProvisionFailed, and we've got people depending on that. I think we need to keep that to limit disruption when we port current openshift-install over to this method. In enhancement I was thinking we prefix everything with Provision but indeed that doesn't matchup as well as ClusterInstall. We could go to new naming and duplicate Failed. WDYT?

Reply:
i can duplicated ClusterInstallFailed to ProvisionFailed ...

Reply:
Any concerns about ClusterInstallInstalled?

Reply:
> Any concerns about ClusterInstallInstalled?

don't particularly like and i prefer the ClusterInstallComplete because ClusteInstall already tells us the action were are trying to Complete and ClusterInstallComplete means we have completed that action.